superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -327,6 +330,41 @@ class SliceHeaderControls extends React.PureComponent<
           </Menu.Item>
         )}
 
+        {this.props.supersetCanExplore && (
+          <Menu.Item key={MENU_KEYS.VIEW_RESULTS}>
+            <ModalTrigger
+              triggerNode={
+                <span data-test="view-query-menu-item">
+                  {t('View results')}
+                </span>
+              }
+              modalTitle={t('Chart Data: %s', slice.slice_name)}
+              modalBody={
+                <ResultsPane
+                  queryFormData={this.props.formData}
+                  queryForce={false}
+                  dataSize={20}
+                  isRequest
+                />
+              }
+              modalFooter={
+                <>
```

Comment:
I think that React.Fragment is not necessary

Reply:
nice catch! Thanks!